.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,64 @@
+---
+name: Bug Report
+about: Help us understand what we need to fix.
+title: ''
+labels: ''
+assignees: ''
+---
+
+<!-- Before submitting this issue, please search and make sure it hasn't already been reported, thanks! -->
```

Comment:
GitHub will suggest possible existing issues as they type in their details, so I don't think we need this.

Reply:
Okay